submissions/OgeEjim/DOM/newsletter.js

```diff
@@ -2,12 +2,12 @@ let submitNewsletter = document.getElementById('valForm');
 let mail = document.getElementById('mail');
 
 function mailValidation(mail,event) {
```

Comment:
No need passing `mail` as a parametr

Reply:
Okay ma,  noted! I've also excluded it as a parameter.